my-playbook/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Install Nginx
+  apt:
+    name: "{{ nginx_packages }}"
+    state: present
+  become: true
+  tags: ['nginx']
+
+- name: Copy Nginx configuration
+  template:
+    src: "{{ nginx_template_file }}"
+    dest: "/etc/nginx/sites-available/default"
+    mode: 0644
+  notify: restart nginx
```

Comment:
this one too, is it required ?

Reply:
Yes, Both handlers are required because updating either the Nginx configuration file or the soft link requires a restart of the Nginx service to apply the changes.